pkg/lib/config/secret.go

```diff
@@ -86,26 +86,33 @@ func (c *SecretConfig) Overlay(layers ...*SecretConfig) *SecretConfig {
 	return merged
 }
 
-func (c *SecretConfig) Lookup(key SecretKey) (*SecretItem, bool) {
-	for _, item := range c.Secrets {
+func (c *SecretConfig) Lookup(key SecretKey) (*SecretItem, bool, int) {
```

Comment:
If there is a bool in the return value to represent ok or not, the convention is to make it the last element in the return value tuple. So the return type should be `(int, *SecretItem, bool)`. Index is the first item so that it is consistent with range loop.

Reply:
Fixed